lib/travis/build/script/r.rb

```diff
@@ -81,7 +81,9 @@ def configure
                 end
 
                 # Both c2d4u and c2d4u3.5 depend on this ppa for ffmpeg
-                sh.cmd 'sudo add-apt-repository -y "ppa:kirillshkrogalev/ffmpeg-next"'
+                if config[:os] == 'trusty'
```

Comment:
I don't think `config[:os]` will ever be set to `trusty`. I don't think anything will, either.

At run time, you can test with `$(lsb_release -cs) = 'trusty'`:

```ruby
sh.if "$(lsb_release -cs) = 'trusty'" do
  sh.cmd 'sudo add-apt-repository -y "ppa:kirillshkrogalev/ffmpeg-next"'
end
```

Reply:
Change incorporated! Thanks for the suggestion